modules/virt-cnv-storage-alerts.adoc

```diff
@@ -0,0 +1,377 @@
+          url: docker://.../cdi-func-test-tinycore
+      storage:
+        resources:
+          requests:
+            storage: 5Gi
+        storageClassName: rook-ceph-block
+----
+
+Otherwise, open an issue and provide the information gathered in the diagnosis process.
+
+//CDIDataVolumeUnusualRestartCount Alert
+== CDIDataVolumeUnusualRestartCount alert
+
+.Meaning
+
+The `.RestartCount` field for a DataVolume shows the number of times that a CDI ephemeral workload pod restarts.
+
+For example, when an HTTP import occurs, the `.RestartCount` field indicates the amount of times the CDI importer pod restarts.
+
+This alert occurs if any DataVolume's restart count is greater than three.
+
+.Impact
+
+If the restart count is greater than three, the DataVolume fails to create a VM on a PVC.
+
+.Diagnosis
+
+* Find the erroneous DataVolume’s name and namespace.
++
+[source,terminal]
+----
+oc get dv -A -o json | jq -r '.items[] | select(.status.restartCount>3)' | jq '.metadata.name, .metadata.namespace'
+----
+
+* Change the `<dv_name>` and `<dv_namespace>` fields based on the erroneous DV's name and namespace to find the corresponding worker pod.
++
+[source,terminal]
+----
+oc get pods -n <dv_namespace> -o json | jq -r '.items[] | select(.metadata.ownerReferences[] | select(.name=="<dv_name>")).metadata.name'
+----
+
+* Change the `<worker_pod>` field that corresponds to the erroneous DV's namespace to check the failing deployments’ corresponding description and logs.
++
+[source,terminal]
+----
+oc -n <dv_namespace> describe pods <worker_pod>
+
+oc -n <dv_namespace> logs <worker_pod>
+----
+
+.Mitigation
+
+In some cases, the error is an incorrect URL, indicated by a 404 error when diagnosing the problem. If an incorrect URL is the cause, you can restart by deleting the DV, correcting the URL on the DV manifest, and recreating the DV.
+
+Open an issue and provide the information gathered in the diagnosis process.
+
+//CDINotReady Alert
+== CDINotReady alert
+
+.Meaning
+
+If a CDI installation is in a degraded state, then the installation is not progressing or not available to use.
+
+.Impact
+
+If the CDI is unusable, then you cannot build Virtual Machine Disks on PVCs using CDI’s DataVolumes (DVs). Additionally, components are not ready and stop progressing towards a ready state.
+
+.Diagnosis
+
+* Check the cdi-operator’s pod namespace.
++
+[source,terminal]
+----
+export openshift-cnv="$(oc get deployment -A | grep cdi-operator | awk '{print $1}')"
+----
+In this example, the kind of operand is `kubevirt` and the operand's name is `kubevirt-kubevirt-hyperconverged`.
+
+* Verify if any of the CDI components are currently not ready.
++
+[source,terminal]
+----
+oc -n $openshift-cnv get deploy -l cdi.kubevirt.io
+----
+
+* Check the failing deployments’ corresponding pod logs and describe (substitute).
++
+[source,terminal]
+----
+oc -n $openshift-cnv describe pods <corresponding_pod_name>
```

Comment:
@mgarrellRH , Lets revert this and keep the $NAMESPACE instead of $openshift-cnv